### PR TITLE
Google profile image

### PR DESCRIPTION
**Pull request by christophgysin**
*Issue #, if available:*
Issue #2466 

*Description of changes:*
Return profile image url for Google authentication provider.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jordanranz** [comment]
I think this implementation work for now and solves your use case. 

In the future maybe we should expose the whole user object returned from the third party auth provider so we don't have to maintain our own copy of it.

**Comment by christophgysin** [comment]
Abstracting the third party object has the advantage that all providers return e.g. the profile picture under the same property.

File: packages/aws-amplify-react/src/Auth/Provider/withGoogle.jsx [hunk 1]
```diff
@@ -68,7 +68,8 @@ export default function withGoogle(Comp) {
             const profile = googleUser.getBasicProfile();
             let user = {
                 email: profile.getEmail(),
-                name: profile.getName()
+                name: profile.getName(),
+                image: profile.getImageUrl(),
```
Comment (jordanranz) [inline packages/aws-amplify-react/src/Auth/Provider/withGoogle.jsx]: Could you change 'image' to 'picture'. I think we would like to stay consistent between providers for this and it seems Auth0 and Facebook both use 'picture'.

**Comment by jordanranz** [comment]
LGTM after the one in-line comment I made.

**Comment by jordanranz** [comment]
> Abstracting the third party object has the advantage that all providers return e.g. the profile picture under the same property.

Agreed, maybe it would be exposed as a separate object so that devs using Amplify can access properties that are uncommon. I'll bring it up to the team in a design review.

File: packages/aws-amplify-react/src/Auth/Provider/withGoogle.jsx (hunk 1 above)
Comment (christophgysin) [reply packages/aws-amplify-react/src/Auth/Provider/withGoogle.jsx]: Done.

**Pull request merged**